.travis.install.cassandra.sh

```diff
@@ -22,7 +22,7 @@ set -xe
 cd "${HOME}"
 
 C_MAJOR="3"
-C_MINOR="7"
+C_MINOR="9"
```

Comment:
Ah, sorry, I wasn't paying attention to that this was doing before.

@jsanda is there any reason why we are using 3.9 here instead of 3.0.x? And another question, if this is based on release/0.18.0, shouldn't we be using Cassandra 2.2.x?